index.js

```diff
@@ -4,6 +4,7 @@ var path = require('path');
 
 // vendor
 var resv = require('resolve');
+var process = require('process');
```

Comment:
this is very much not needed, since it's a node global, and this is a node module. It's a bundler's job to shim node globals when bundling node modules.

Reply:
see https://github.com/browserify/browser-resolve/issues/99#issuecomment-770465307